src/chess/uciloop.cc

```diff
@@ -200,6 +201,21 @@ void UciLoop::SendResponse(const std::string& response) {
   std::cout << response << std::endl;
 }
 
+#define STR_INNER(a) #a
+#define STR(a) STR_INNER(a)
```

Comment:
There's still a good chance, in general, that you're accidentally overwriting some other macro with this (and #undef doesn't help, since it can't restore *previous* macros). Though I guess it's not too much of a problem for a non-library like lc0.

Reply:
personally don't care but seems like a lot of macro magic, why not just
const static char version[] = "0.13.0";  or whatever

if you really want the macros maybe just add:
#ifdef STR_INNER
#error "oh no"
#endif 

etc